crates/forge/bin/cmd/compiler.rs

```diff
@@ -94,13 +94,17 @@ impl ResolveArgs {
                         })
                         .collect();
 
-                    (version.clone(), paths)
+                    (
+                        version.clone(),
+                        EvmVersion::default().normalize_version_solc(version).unwrap_or_default(),
+                        paths,
+                    )
```

Comment:
More of a general note / question, should `--json` output adhere to the verbosity flag? If not, should we mark the verbosity flag to be at conflict with `--json`?

Reply:
yeah, I think it should in order to be consistent with other commands, like in forge test, will adapt